internal/website/content/howto/blob/open-bucket.md

```diff
@@ -0,0 +1,328 @@
+---
+title: "Open a Bucket"
+date: 2019-03-20T14:51:29-07:00
+draft: true
+weight: 1
+---
+
+The first step in interacting with unstructured storage is connecting to your
+storage backend. Every storage provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*blob.Bucket` type][].
+
+[`*blob.Bucket` type]: https://godoc.org/gocloud.dev/blob#Bucket
+
+## Constructors versus URL openers
+
+If you know that your program is always going to use a particular backend
+storage type or you need fine-grained control over the connection settings,
+you should call the constructor function in the driver package directly (like
+`s3blob.OpenBucket`). However, if you want to change backends based on
+configuration, you can use `blob.OpenBucket`, making sure you ["blank import"][]
+the driver package to link it in. This guide will show how to use both forms for
+each storage provider.
+
+["blank import"]: https://golang.org/doc/effective_go.html#blank_import
+
+## S3
```

Comment:
I would put the URLs first and de-emphasize the constructors as "advanced use". WDYT? I personally found URLs to be much more usable, and I'd imagine they're sufficient for most users so it would be nice to start with something that's easier to grok and occupies fewer LOCs